src/Symfony/Component/Validator/Constraints/JwtValidator.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+/**
+ * @author Mokhtar Tlili <tlili.mokhtar@gmail.com>
+ */
+class JwtValidator extends AbstractFormatValidator
+{
+    protected function validateFormat(string $value): bool
+    {
+        // Split the JWT into its three parts
+        $parts = explode('.', $value);
+
+        // JWT should have exactly three parts
+        if (count($parts) !== 3) {
+            return false;
+        }
+
+        [$header, $payload, $signature] = $parts;
```

Comment:
$signature is unused
```suggestion
        [$header, $payload] = $parts;
```